src/form/form-field.ts

```diff
@@ -20,3 +90,5 @@ export interface FormFieldItem {
     value: string;
     label: string;
 }
+
+export default FormField;
```

Comment:
Might be better to export directly in the `FormField` atop.

```ts
export type FormField = SystemFormField | CustomFormField;
```

Reply:
I still want to keep FormField as the default export, can't do it in one line.

Reply:
oh, my bad, I missed the default...